config/routes.rb

```diff
@@ -7,6 +7,7 @@
   namespace :account do
     resources :companies do
       resources :pages, only: %i[show index]
+      resource :download, only: [:show]
```

Comment:
why resource? it's a lot easier to use member action on companies controller
http://guides.rubyonrails.org/routing.html#adding-member-routes